database/src/test/kotlin/no/nav/su/se/bakover/database/TestDataHelper.kt

```diff
@@ -113,4 +82,24 @@ internal class TestDataHelper(
         ).also {
             revurderingRepo.lagre(it)
         }
+
+    fun uavklartVilkårsvurdering(
```

Comment:
Todo; kunne sikkert gjort seg med litt mer test-support for å lage db-tester. Litt tungvindt at man ikke kan gjøre direkte insert av den typen man vil ("senere" tilstander oppdaterer bare spesifikk data). Kan kanskje gjøre det litt hyggeligere å skrive tester om vi lager noe bedre support knyttet til det.